src/vsr/replica.zig

```diff
@@ -4743,16 +4743,36 @@ pub fn ReplicaType(
                 return true;
             }
 
-            if (!message.header.operation.valid(StateMachine)) {
```

Comment:
I think we still need to check this first. `StateMachine.operation_from_vsr` will panic (by way of `vsr.Operation.to(StateMachine, operation)` -> `assert(operation.valid(StateMachine));`), so the `ignoring invalid operation` is unreachable.